core/src/App.html

```diff
@@ -1,16 +1,16 @@
 
 <div id="app {hideNav? 'no-nav' : ''}">
-<Backdrop>
+  <Backdrop>
     <div class="fd-page iframeContainer" use:init="context"></div>
-    {#if showLoadingIndicator}
-    <div in:fade='{delay: 250, duration: 250}' out:fade='{duration: 250}' class="fd-page spinnerContainer" aria-hidden="false" aria-label="Loading">
-        <div class="fd-spinner"><div></div></div>
-    </div>
-    {/if}
-    
-    <TopNav pathData={navigationPath} />
-    <LeftNav pathData={navigationPath} />
-    </Backdrop>
+  </Backdrop>
```

Comment:
I tested all those options. The old implementation was not working as expected, none of the elements was inside the backdrop. Apart from that, you need to have the backdrop at the same DOM level as the top and left nav, so that it covers the elements properly. If you had those navs inside the backdrop container, you would have displaying issues due to how we display the different containers. Additionally, having the iframe container inside the backdrop is IMHO a good approach, because it allows you to apply the iframe on top of the backdrop without using any z-index.

Reply:
I didn't see any unexpected behaviour but if you did, let's leave it like this.